src/main/java/core/basesyntax/dao/impl/UserDaoImpl.java

```diff
@@ -12,21 +14,44 @@ public UserDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public User create(User entity) {
-        return null;
+        Transaction transaction = null;
+        Session session = null;
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            session.save(entity);
+            transaction.commit();
+            return entity;
+        } catch (Exception e) {
+            throw new RuntimeException("Can`t create entity", e);
+        } finally {
+            if (session != null) {
+                session.close();
+            }
+        }
     }
 
     @Override
     public User get(Long id) {
-        return null;
+        try (Session session = factory.openSession()) {
+            User user = session.get(User.class, id);
+            System.out.println(user.getComments());
```

Comment:
do we need sout here?

Reply:
don`t know exact reason, but yes, test "GetById_WithComments_Ok" in UserDaoImplTest fails without user.getComments().toString();

Reply:
yse, because you shulld put FetchType.EAGER in User class